packages/teams-js/src/public/meeting.ts

```diff
@@ -435,19 +454,35 @@ export namespace meeting {
 
   /**
    * Registers a handler for changes to the selfParticipant raiseHand state. If the selfParticipant raises their hand, isHandRaised
-   * will be true. By default and if the selfParticipant hand is lowered, isHandRaised will be false. This API will return {@link IRaiseHandStateChangedEvent}
+   * will be true. By default and if the selfParticipant hand is lowered, isHandRaised will be false. This API will return {@link RaiseHandStateChangedEvent}
    * that will have the raiseHand state or an error object. Only one handler can be registered at a time. A subsequent registration
    * replaces an existing registration.
    *
    * @param handler The handler to invoke when the selfParticipant's raiseHand state changes.
    */
   export function registerRaiseHandStateChangedHandler(
-    handler: (raiseHandStateChangedEvent: IRaiseHandStateChangedEventData) => void,
+    handler: (eventData: RaiseHandStateChangedEventData) => void,
   ): void {
     if (!handler) {
       throw new Error('[registerRaiseHandStateChangedHandler] Handler cannot be null');
     }
     ensureInitialized(FrameContexts.sidePanel, FrameContexts.meetingStage);
     registerHandler('meeting.raiseHandStateChanged', handler);
   }
+
+  /**
+   * Registers a handler for receiving meetingReactions. When the selfParticipant successfully sends a meetingReaction and it is being rendered on the UI, the meetingReactionType will be populated. Only one handler can be registered
```

Comment:
what's `selfParticipant`?

Reply:
selfParticipant is the _current user for the your instance_. Meaning if you and I are in a meeting. For my instance of the code, I am the selfParticipant and you are just another participant. Vice versa when looking from your perspective. 


Regarding camelCase language, I can change that.

Reply:
That's great, but once you merge the change, and people only see the code/comment how would they know what `selfParticipant` is and all the other things you just typed here to answer my question? I think we need to rework the comment to include some of that extra context for future reference.